spec/lexers/raml_spec.rb

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*- #
+
+describe Rouge::Lexers::RAML do
+  let(:subject) { Rouge::Lexers::RAML.new }
+
+  describe 'guessing' do
+    include Support::Guessing
+
+    it 'guesses by filename' do
+      assert_guess :filename => 'bar.raml'
+    end
+
```

Comment:
Drop this spec in here, too:

```
    it 'guesses by mimetype' do
      assert_guess :mimetype => 'application/raml+yaml'
    end
```

Be sure to clean up whitespace/newlines so there aren't multiple.

Reply:
Whoops! I forgot that this spec is also useful since you added the detect for the `#%RAML 1.0` at the top of the file. Something like:

```
it 'guesses by source' do
  assert_guess :source => '#%RAML 1.0'
end
```

Reply:
The spec needs to have the checks @dblessing mentioned above.